src/app/submit-item/components/submit/submit.component.scss

```diff
@@ -24,4 +24,11 @@ div.confirm {
   display: flex;
   flex-direction: column;
   align-items: center;
+
 }
+
+
+::ng-deep .mat-checkbox-layout {
+  white-space: normal !important;
+}
```

Comment:
::ng-deep and !important should be used as little as possible, as they overwrite the styles for all elements of that class in the entire Angular app. 

Why do you need to modify the white-space property of the mat-check-box layout?
Is there any other way to achieve it?

Reply:
habe es momentan auskommentiert, da wir spätestens für kleinere devices eine Art word-wrap benutzen müssen, da sonst lange Sätze komischerweise bei einigen Material Komponenten nicht in mehreren Zeilen komprimiert werden.